sample_caller.py

```diff
@@ -190,6 +289,29 @@ def main():
             "wpd, and doc files."
         ),
     )
+    parser.add_option(
+        "--extract-content",
+        action="store_true",
+        default=False,
+        help=(
+            "Extract document's content using `doctor`. "
+            "Then, execute Site.extract_from_text method. "
+            "If this flag is set to True, it will "
+            "make `binaries` True, too. This requires "
+            "having a running `doctor` instance"
+        ),
+    )
+    parser.add_option(
+        "--doctor-host",
+        default=os.environ.get("JURISCRAPER_DOCTOR_HOST", ""),
```

Comment:
I think doctor when from from the repo - defaults to `0.0.0.0:5050` 

what do you think about 
`default=os.environ.get("JURISCRAPER_DOCTOR_HOST", "0.0.0.0:5050"),`

Reply:
Sure, it can be set to that value

I am pairing this with your other comment:

> Would just running docker-compose -f docker-compose.dev.yml up be an easier options that installing requirements and than run server ?

If the container accepts requests it should work out of the box by using it's host

About what's easier, it depends. I just keep having problem with docker networking due to using a VPN, so for me it's easier to use `runserver`